packages/components/src/button/style.scss

```diff
@@ -11,7 +11,7 @@
 	&.is-button {
 		padding: 0 10px 1px;
 		line-height: 26px;
-		height: 28px;
```

Comment:
These button styles are duplicated from core, so they should be updated there too.